indico/modules/events/abstracts/templates/abstract/review.html

```diff
@@ -74,280 +17,43 @@
-                                <div class="abstract-comment-badges">
-                                    <a class="i-button-icon icon-edit js-edit-comment"
-                                       title="{% trans %}Edit comment{% endtrans %}"
-                                       data-href="{{ url_for('.edit_abstract_comment', comment) }}"
-                                       data-form-container="#comment-{{ comment.id }}"
-                                       data-update=".management-page"
-                                       data-replace-update
-                                       data-confirm-close-unsaved
-                                       data-ajax-form></a>
-                                    <a class="i-button-icon icon-cross js-delete-comment"
-                                       title="{% trans %}Remove comment{% endtrans %}"
-                                       data-method="DELETE"
-                                       data-href="{{ url_for('.delete_abstract_comment', comment) }}"
-                                       data-title="{% trans %}Remove comment{% endtrans %}"
-                                       data-confirm="{% trans %}Are you sure you want to remove this comment?{% endtrans %}"
-                                       data-ajax>
-                                    </a>
-                                </div>
-                            {% endif %}
-                        </div>
-                        <div class="i-box-content" id="comment-{{ comment.id }}">
-                            {{ comment.text }}
-                        </div>
-                    </div>
-                </div>
-            {% endfor %}
-            {% if can_comment %}
-                {{ render_comment_box(abstract, form) }}
-            {% endif %}
-        </div>
-    {% endif %}
-{% endmacro %}
-
-{% macro render_comment_box(abstract, form) %}
-    <div id="abstract-comment-box" class="i-timeline-item">
-        <div class="i-timeline-item-label action icon-file"></div>
-        <div class="i-timeline-item-box footer-only header-indicator">
-            <div class="i-box-footer js-form-container">
-                {{ render_comment_form(form, abstract) }}
-            </div>
-        </div>
-    </div>
-{% endmacro %}
 
 {% macro render_comment_form(form, abstract, comment=none) %}
     {% set url = url_for('.edit_abstract_comment', comment) if comment else url_for('.comment_abstract', abstract) %}
     {% set form_class = 'edit-comment' if comment else 'new-comment unfocused' %}
     {% call form_header(form, action=url, orientation='vertical', classes=form_class) %}
-        data-form-container="#abstract-comment-box .js-form-container"
-        data-update=".management-page"
-        data-replace-update
-        data-ajax-form
+        {% if comment %}
+            data-form-container="#abstract-comment-{{ comment.id }} .js-form-container"
```

Comment:
`data-form-container` without `data-ajax-form` doesn't do anything so this line seems to be useless